rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -91,6 +92,7 @@ std::vector<std::string> GetAdapters()
 SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 	: wxDialog(parent, wxID_ANY, "Settings", wxDefaultPosition)
 {
+	SetIcon(wxGetApp().m_MainFrame->GetIcon());
```

Comment:
Sure ? i juste think is more godd look ^^
![capture d ecran 2015-12-19 18 15 39](https://cloud.githubusercontent.com/assets/5261759/11914278/8acaec9a-a67c-11e5-98bd-78dc96fd3c92.png)